cli/http_body.rs

```diff
@@ -1,15 +1,94 @@
 // Copyright 2018-2019 the Deno authors. All rights reserved. MIT license.
```

Comment:
This file still needs cleanup to get rid of Hyper related stuff, I want to get CI green first tho